web/src/components/profile/ProfileTab.tsx

```diff
@@ -1,38 +1,23 @@
 import { Icon } from "@/components/njwds/Icon";
-import { useConfig } from "@/lib/data-hooks/useConfig";
 import { ProfileTabs } from "@/lib/types/types";
 import { ReactElement } from "react";
 
 interface Props {
   tab: ProfileTabs;
   activeTab: ProfileTabs;
   setProfileTab: (profileTab: ProfileTabs) => void;
+  children: string;
```

Comment:
[pebble] I don't like calling a single string `children`. This seems like an inconsistent use of a prop that is used commonly in React. Additionally, this is a plural name that is representing a single value.

Reply:
updated to tabText